api-auto-registration/key-concepts.hbs.md

```diff
@@ -199,26 +201,26 @@ metadata:
 spec:
   type: openapi         # type of the API spec. oneOf(openapi, grpc, asyncapi, graphql)
   title:                # title of the curated API
-  description:          # description of the curated API
-  documentation:        # TODO
+  description:          # description of the curated API spec
+  documentation:        # documentation for the curated API spec
   groupId:              # groupID of the curated API.
   version:              # version of the curated API
   apiDescriptors:
     - name:             # name of a APIDescriptor to include in this curated API
-      namespace:        # optional namespace of the APIDescriptor
+      namespace:        # namespace of the APIDescriptor
```

Comment:
```suggestion
      namespace:        # (optional) namespace of the APIDescriptor
```

Reply:
Could you also find the other optional fields in the docs and make sure they are following the same style (upper case vs. lower case).

Reply:
So the namespace is currently not optional https://gitlab.eng.vmware.com/tap-api-first/api-auto-registration/-/blob/main/controller/api/v1alpha1/curatedapidescriptor_types.go#L47

```
	Namespace string `json:"namespace"`
```

I couldn't remember if it was decided one way or another. Does this need to be fixed in our codebase then?

Reply:
Oh wow good catch! Let's leave it for now for explicity. It's easier to make a required field optional but not the other way around. Let's see if there are users complaining about it.